code/obj/item/grenades.dm

```diff
@@ -1202,6 +1203,7 @@ PIPE BOMBS + CONSTRUCTION
 				name = "pipe bomb frame"
 
 			desc = "Two small pipes joined together, filled with welding fuel and connected with a cable. It needs some kind of ignition switch."
```

Comment:
```suggestion
			desc = "Two small pipes joined together, filled with explosives and connected with a cable. It needs some kind of ignition switch."
```
more generic

Reply:
Yeah, that is a valid point, I saw that but didn't consider I could fix it. >.<